etcdserver/etcdserverpb/rpc.proto

```diff
@@ -658,6 +665,26 @@ message LeaseKeepAliveResponse {
   int64 TTL = 3;
 }
 
+message LeaseTimeToLiveRequest {
+  // ID is the lease ID for the lease.
+  int64 ID = 1;
+  // keys is true to query all the keys attached to this lease.
+  bool keys = 2;
+}
+
+message LeaseTimeToLiveResponse {
+  ResponseHeader header = 1;
+  // ID is the lease ID from the keep alive request.
+  int64 ID = 2;
+  // TTL is the remaining TTL in seconds for the lease.
+  // Milliseconds will be rounded off.
+  int64 TTL = 3;
```

Comment:
@heyitsanthony i belive yours is better. we do not really expose ms accuracy anyway. we want users to understand a 0 TTL does not mean the lease is expired.